packages/protocol/contracts/common/UsingPrecompiles.sol

```diff
@@ -16,6 +16,14 @@ contract UsingPrecompiles {
   address constant GET_PARENT_SEAL_BITMAP = address(0xff - 10);
   address constant GET_VERIFIED_SEAL_BITMAP = address(0xff - 11);
 
+  /**
+  * @notice Returns the storage, major, minor, and patch version of the contract.
+  * @return The storage, major, minor, and patch version of the contract.
+  */
+  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
```

Comment:
ditto
I don't think we want this to be versioned as multiple versioned contracts inherit